src/python/pants/engine/native.py

```diff
@@ -291,8 +291,7 @@ def extern_get_handle_from_type_id(self, context_handle, ty):
 
   @_extern_decl('bool', ['ExternContext*', 'TypeId'])
   def extern_is_union(self, context_handle, type_id):
-    "Return whether or not a type is a member of a union"
-
+    """Return whether or not a type is a member of a union"""
```

Comment:
Too small to burn extra CI for but we tend to end all doc sentences with period.